src/domain/collaboration/callout/whiteboard/WhiteboardCallout.tsx

```diff
@@ -118,6 +118,11 @@ const WhiteboardCallout = forwardRef<HTMLDivElement, WhiteboardCalloutProps>(
                   profile: {
                     id: '',
                     displayName: '',
+                    storageBucket: {
+                      // TODO: When creating a whiteboard a StorageBucketId is needed if we want to allow image uploading
```

Comment:
does this still apply?

Reply:
yes, this is an known issue already. We have to create entities when we open those Create dialogs in order to have an ID and a StorageBucket for storing images for whiteboards and markdowns. 
At the moment the create dialogs store images in the parent SB (creating a challenge and adding an image to any of the markdowns will store the image in the parent space). But once we have the challenge created we have its SB and we can store challenge images directly there.
We have mentioned this but I'm not sure if there is a task already, I'll check it now and create it if not.